KomaMRIPlots/Project.toml

```diff
@@ -21,7 +23,6 @@ KomaPlotsPlutoPlotlyExt = "PlutoPlotly"
 [compat]
 Interpolations = "0.13, 0.14, 0.15"
 Kaleido_jll = "0.1"
-KomaMRIBase = "0.8"
```

Comment:
Revert

Reply:
My bad, how can I keep these lines if I have this package in development mode?

Reply:
```julia
using Pkg
Pkg.develop([
    PackageSpec(path=pwd(), subdir="."),
    PackageSpec(path=pwd(), subdir="KomaMRIBase"),
    PackageSpec(path=pwd(), subdir="KomaMRICore"),
    PackageSpec(path=pwd(), subdir="KomaMRIFiles"),
    PackageSpec(path=pwd(), subdir="KomaMRIPlots")
])
```

Reply:
Do I have to do this from default environment?

Reply:
Resolved in [7f1cc52](https://github.com/JuliaHealth/KomaMRI.jl/pull/184/commits/7f1cc52925d3be8803c4ff0c95c1e06f391b7c83)